src/components/MainPage.js

```diff
@@ -26,15 +32,17 @@ class MainPage extends Component {
   }
   //  Fetches the available currencies from the API before the component loads.
   componentWillMount() {
-    CryptoAPICalls.onAppLoad().then(res => {
-      this.setState({
-        cryptoList: res.cryptoList,
-        intlList: res.intlList,
-        selectedCrypto: res.cryptoList[0].code,
-        selectedIntl: res.intlList[0].code
-      });
+		CryptoAPICalls.onAppLoad().then(res => {
+			this.setState({
+				cryptoList: res.cryptoList,
+				initializing: false,
```

Comment:
`initializing` is set here because it triggers the `LoadingSpinner` to be shown to show the user their initial data is being fetched.